### PR TITLE
gh #260 dsDisplay L1 update

### PR DESCRIPTION
**Pull request by Sumip98**
Added error checking logic in test_l1_dsDisplay_negative_dsGetDisplayAspectRatio

### CODE REVIEW

File: src/test_l1_dsDisplay.c [hunk 1]
```diff
@@ -845,27 +845,29 @@ void test_l1_dsDisplay_positive_dsGetDisplayAspectRatio(void) {
...
- * |05|Call dsGetDisplayAspectRatio() with an NULL aspectRatio | intptr_t handle, NULL | dsERR_INVALID_PARAM | Should return error indicating invalid handle |
- * |06|Terminate the display sub-system with dsDisplayTerm() | | dsERR_NONE | Termination should succeed |
- * |07|Call dsGetDisplayAspectRatio() without initializing the display sub-system or obtaining a handle | intptr_t handle, dsVideoAspectRatio_t *aspectRatio  | dsERR_NOT_INITIALIZED | Should return error indicating the module is not initialized |
+ * |04|If the device is a source, call dsGetDisplayAspectRatio() with an invalid handle| NULL, dsVideoAspectRatio_t *aspectRatio | dsERR_INVALID_PARAM | Should return error indicating invalid handle |
+ * |05|If the device is a source, call dsGetDisplayAspectRatio() with an NULL aspectRatio | intptr_t handle, NULL | dsERR_INVALID_PARAM | Should return error indicating invalid handle |
+ * |06|If the device is a sink, call dsGetDisplayAspectRatio() with a valid handle | intptr_t handle, dsVideoAspectRatio_t *aspectRatio | dsERR_OPERATION_NOT_SUPPORTED | API is not supported for sink devices |
```
Comment (srinivasgtl) [inline src/test_l1_dsDisplay.c:850]: No need to be an valid handle as well.  The API should return UNSUPPORTED if it is not a supported device.

**Review by srinivasgtl — COMMENTED** [review]
Update this one comment.

File: src/test_l1_dsDisplay.c (hunk 1 above)
Comment (Sumip98) [reply src/test_l1_dsDisplay.c]: updated

**Review by hari22yuva — APPROVED** [review]
looks ok

**Review by bhanucbp — APPROVED** [review]
ship it

**Pull request merged**